.gitmodules

```diff
@@ -288,7 +288,6 @@
 [submodule "third_party/libwebsockets/repo"]
 	path = third_party/libwebsockets/repo
 	url = https://github.com/warmcat/libwebsockets
-	platforms = linux,darwin,tizen
```

Comment:
Why this was removed? libwebsockets is required for all platforms?

Reply:
According to `./scripts/run_in_build_env.sh "./scripts/build/build_examples.py targets" | grep chip-tool` these are the platforms that can compile chip tool:

android (but here I don't think web sockets is a thing ... it is a different chip tool and that makes it confusing...), linux, imx, tizen.

Anything else should fix the CI/build scripts and say `checkout_submodules --platform esp32 --platform linux` to checkout everything (I assume that works ... if it does not, we should fix it).

Reply:
Removed this change